github/modals/fileCodeModal.ts

```diff
@@ -67,26 +67,27 @@ export async function fileCodeModal({
 
     block.addDividerBlock();
 
-    block.addActionsBlock({
-        elements: [
-            block.newButtonElement({
-                actionId: ModalsEnum.MERGE_PULL_REQUEST_ACTION,
-                text: {
-                    text: ModalsEnum.MERGE_PULL_REQUEST_LABEL,
-                    type: TextObjectType.PLAINTEXT,
-                },
-                value: room?.id,
-            }),
-            block.newButtonElement({
-                actionId: ModalsEnum.COMMENT_PR_ACTION,
-                text: {
-                    text: ModalsEnum.COMMENT_PR_LABEL,
-                    type: TextObjectType.PLAINTEXT,
-                },
-                value: room?.id,
-            }),
-        ],
-    });
+    // Adding these will complex the user expereince
```

Comment:
don't understand your comment here

Either explain clearly what you mean, or totally remove the code.     It will *always* still be in Github history anyways.

Reply:
I will update this code, its still WIP.